src/pools/abstract-pool.ts

```diff
@@ -171,7 +171,7 @@ export abstract class AbstractPool<
    * @param data The input for the specified task.
    * @returns Promise that will be resolved when the task is successfully completed.
    */
-  public execute (data: Data): Promise<Response> {
+  public execute (data: Data): Promise<unknown> {
```

Comment:
If you merge with master, that change will become unnecessary.

Reply:
I Will give It a try or of you want feel free ti change it

Reply:
@jerome-benoit  I tried to merge with master and remove the unknown in favor of Response type but does not work.
Would you help me with those types please? I am too unfamiliar with TS

Reply:
Nevermind, I think I did it